pages/agent/v3/unclustered_tokens.md

```diff
@@ -3,7 +3,7 @@
 > 🚧 This page documents a deprecated Buildkite feature
 > Previously, agents only connected directly to Buildkite via a token which was created and managed by the processes described on this page. These tokens are now a deprecated feature of Buildkite, and are referred to as _unclustered agent tokens_. Unclustered agent tokens, however, are still available to customers who have not yet migrated their pipelines to a [cluster](/docs/clusters/overview).
 > _Agent tokens_ are now associated with clusters, and connect to Buildkite through a specific cluster within an organization. Learn more about how to manage agent tokens for clusters in [Agent tokens](/docs/agent/v3/tokens) and how to [move your unclustered agents across to a cluster](/docs/clusters/manage-clusters#move-unclustered-agents-to-a-cluster).
```

Comment:
Thanks @HugeIRL for your contribution!

I was just discussing this with @mbelton-buildkite and we thought that since we're starting to use italics to emphasise new/key terms and statements, which is more commonly used convention in documentation, that we'd reformat this to _italics_ rather than **bold**.

Also, we've moved this sentence to the top of this note.

```suggestion
> 🚧 This page documents a deprecated Buildkite feature
> _It is not be possible to create and work with unclustered agents for any new Buildkite organizations created after the official release of clusters on February 26, 2024._ Therefore, unclustered agent tokens are not relevant to these organizations.
> Previously, agents only connected directly to Buildkite via a token which was created and managed by the processes described on this page. These tokens are now a deprecated feature of Buildkite, and are referred to as _unclustered agent tokens_. Unclustered agent tokens, however, are still available to customers who have not yet migrated their pipelines to a [cluster](/docs/clusters/overview).
> _Agent tokens_ are now associated with clusters, and connect to Buildkite through a specific cluster within an organization. Learn more about how to manage agent tokens for clusters in [Agent tokens](/docs/agent/v3/tokens) and how to [move your unclustered agents across to a cluster](/docs/clusters/manage-clusters#move-unclustered-agents-to-a-cluster).
```